CMakeLists.txt

```diff
@@ -179,6 +179,45 @@ if (PYBIND11_TEST)
   add_subdirectory(tests)
 endif()
 
+include(GNUInstallDirs)
+include(CMakePackageConfigHelpers)
+
+if(CMAKE_VERSION VERSION_GREATER 3.0.1)
+  # Build an interface library target:
+  #   Though any project using pybind11 will need to include python headers and
+  #   define c++11/14-ness, these are not included as interface includes or compile
+  #   defs because (1) that's really the domain of the upstream project to decide,
+  #   (2) we can get away with it since installation (headers) don't care about
+  #   python version or C++ standard, and (3) this will make the easily target relocatable.
+  add_library(pb11 INTERFACE)
```

Comment:
Sure, done. It was just a dummy name so I didn't claim `pybind11` as a target name that you might have wanted to use.